starport/cmd/cmd.go

```diff
@@ -245,3 +247,25 @@ func checkNewVersion(ctx context.Context) {
 
 `, next)
 }
+
+// newApp create a new scaffold app
+func newApp(appPath string, spinner *clispinner.Spinner) (scaffolder.Scaffolder, error) {
+	sc, err := scaffolder.App(appPath)
+	if err != nil {
+		return sc, err
+	}
+
+	if sc.Version.LT(cosmosver.StargateFortyThreeVersion) {
```

Comment:
Should we check for 0.43 or 0.44? I haven't checked if there are breaking changes between these two, if so, better to change to 0.44. /cc @Pantani

Reply:
From the release notes and changelog perspective, I don't see any breaking change from `0.43` to `0.44`. The condition uses the 0.43 because the breaking changes start in this version, and `0.43` is discontinued. It seems the user only need to update the `go.mod` from `0.43` to `0.44`